src/schema/v2/home/__tests__/home_page_fairs_module.test.js

```diff
@@ -1,40 +1,77 @@
-/* eslint-disable promise/always-return */
 import { runQuery } from "schema/v2/test/utils"
+import { nanoid } from "nanoid"
+import moment from "moment"
+import { range } from "lodash"
+
+// adding this dep explicitly so the tests run when we change the main file
+;() => {
+  require("../home_page_fairs_module")
+}
+
+const mockRunningFair = () => {
+  const id = nanoid()
+  return {
+    id: `running-fair-${id}`,
```

Comment:
Maintainability concern: 

Given each of these three fake fairs have a different prefix in their name (`running-fair-`, `future-fair-`, `past-fair`), do we need to append a unique ID? Dynamically generating these IDs and names seems like it would cause confusion when tests fail and we can't find the exact ID or name in the test code.

The only assertions I see that check uniquely identifying features of each fake fair are checking the prefix _without_ the appended ID, which makes me feel more confident they're not needed. 

Maybe there is some other advantage to generating IDs with nanoid -- if so, let me know!

Reply:
The idea was that I can do `fairs = [mockRunningFair(), mockRunningFair()]` and get two different fairs that are both running. Then I could test the output using the first ones as a thing to compare with.

In this case here there was not need for this, true. I guess it could be done either with the `Math.random` that @dzucconi suggested or even just using a global (for the file) counter.

I might have been a bit too overzealous 😬.